cmake/ExtendBuildTypes.cmake

```diff
@@ -6,6 +6,9 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE "Release" CACHE STRING "CMake build type" FORCE)
 endif()
 
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
Global C++ version is good. I had trouble finding this 👍

Reply:
I don't know if this is the right place to put it, if you have a better idea, I am all ears :)

Reply:
I was thinking at the start of the general cmake file, but this is also fine.